### PR TITLE
update: insert "create service" docs in get started pages

### PR DESCRIPTION
**Pull request by ArthurFlag**

### CODE REVIEW

**Review by harshini-rangaswamy — APPROVED** [review]
LGTM!

File: static/includes/create-service-console.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+   :::note
+   This determines the number of servers and the memory,
+   CPU, and disk resources allocated to your service. See
+   [Plans & Pricing](https://aiven.io/pricing).
+   :::
+
+1. Optional: Add [disk storage](/docs/platform/howto/add-storage-space).
+
+1. Enter a name for your service.
```
Comment (harshini-rangaswamy) [inline static/includes/create-service-console.md:28]: Is there a way to hide this step for Flink and Metrics, as you cannot fork these services?

File: static/includes/create-service-console.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+1. Log in to [Aiven Console](https://console.aiven.io/) and open your project.
+
+1. From your project, on the left-side menu, click <ConsoleLabel name="services"/> > **Create service**.
```
Comment (wojcik-dorota) [inline static/includes/create-service-console.md:5]: nitpick: when you open your project, the **Services** page loads by default, no need to select it.

File: static/includes/create-service-console.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+   :::important
+   You cannot change the name after you create the service. You can
+   [fork the service][fork] with a new name instead.
+   :::
+
+1. Optional: Add [tags](/docs/platform/howto/tag-resources).
+
+1. Click **Create service**.
+
+The new service opens on the **Overview** page, which shows
```
Comment (wojcik-dorota) [inline static/includes/create-service-console.md:39]:
```suggestion
The new service opens on the <ConsoleLabel name="overview"/> page, which shows
```

File: static/includes/cli-help.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+To see:
+
+- A full list of default flags, run: `avn service create -h`
+- Type-specific options, run: `avn service types -v`
```
Comment (wojcik-dorota) [inline static/includes/cli-help.md:4]: no need for more colons here?
```suggestion
- A full list of default flags, run `avn service create -h`
- Type-specific options, run `avn service types -v`
```

**Pull request merged**